rest/route/agent_test.go

```diff
@@ -820,3 +820,68 @@ func TestCreateGitHubDynamicAccessToken(t *testing.T) {
 		})
 	}
 }
+
+func TestRevokeGitHubDynamicAccessToken(t *testing.T) {
+	route := "/task/%s/github_dynamic_access_token"
+	taskID := "taskID"
+	token := "some-token-would-be-here"
```

Comment:
nit: L825-827 could be `const` just to be extra sure that none of the test cases accidentally modify them for later tests.